src/Core/Grid/Action/Bulk/Type/SubmitBulkAction.php

```diff
@@ -54,9 +54,12 @@ protected function configureOptions(OptionsResolver $resolver)
             ->setDefaults([
                 'confirm_message' => null,
                 'submit_method' => 'POST',
+                'route_params_extra' => [],
```

Comment:
I understand your need to pass extra params however I'm a bit worried whenever in a software we start adding "extra" things 😅 . Most of the time - not always - it's a shortcut solution, not the best one.

@mickaelandrieu @sarjon an opinion ? 😄

Reply:
yeah - I used it a lot since for my case , its mandatory to pass an extra id to all routes I am generating - I'm thinking now that I can just rename it to `record_route_params` and in my scenario, instead of passing integer value I can pass associate column which is called `id_parent`.

The sample of such column would be `PositionColumn` https://github.com/PrestaShop/PrestaShop/blob/develop/src/Core/Grid/Column/Type/Catalog/PositionColumn.php#L65

what do you think about that ? :)

Reply:
resolved - `route_params_extra` is no longer in use